infra/app/envs/dev/outputs.tf

```diff
@@ -5,3 +5,8 @@ output "account_id" {
 output "region" {
   value = data.aws_region.current.name
 }
+
+output "dns_name" {
```

Comment:
let's think of a different name for this variable. while `alb.dns_name` has a clear meaning, this output is in the `dev` module and it's not clear to me what `dev.dns_name` means, why would an environment have a dns name.